scripts/process-spectrum-postcss-plugin.js

```diff
@@ -401,6 +401,11 @@ class SpectrumProcessor {
             ],
         });
 
+        root.walkAtRules((atRule) => {
```

Comment:
@cuberoot this _seems_ like all the magic needed to get the `@keyframes` rules in, since they are pretty contained to feature specific syntax inside of themselves. The `@media` issue that I added would need to step into the styles that are shipped to ensure that they are required and processed correctly, which is why I left them out for right now.